lib/improver/cli/__init__.py

```diff
@@ -144,6 +145,27 @@ def inputcube(to_convert):
     return maybe_coerce_with(load_cube, to_convert)
 
 
+@value_converter
+def inputvector(to_convert):
+    """Loads 2 vector cubes from a cubelist
+
+    Args:
+        to_convert (string or iris.cube.Cube):
+            File name or Cube object.
+
+    Returns:
+        Loaded cube or passed object.
+
+    """
+    from improver.utilities.load import load_cube
+
+    speed_cube = maybe_coerce_with(load_cube, to_convert,
+                                   constraints="wind_speed")
+    direction_cube = maybe_coerce_with(load_cube, to_convert,
+                                       constraints="wind_from_direction")
+    return CubeList([speed_cube, direction_cube])
```

Comment:
I don't think we want to have something wind specific leaking into this file - we should try and keep it generic.

Maybe we could have something like a `create_constrainedinputcube_converter`, which is a higher order function which returns converter which applies a given constraint? (cc @TomekTrzeciak, who might hate this idea)

Something like...
```
def create_constrainedinputcube_converter(constraints):
    def constrainedinputcube_converter(to_convert):
        return maybe_coerce_with(load_cube, to_convert,
                                                   constraints=constraints)
    return constrainedinputcube_converter
```

Then in `wind_vector_to_uv.py`, you could have something like
```
inputspeedcube = cli.create_constrainedinputcube_converter('wind_speed')
inputdircube = cli.create_constrainedinputcube_converter('wind_from_direction')
def process(wind_speed: inputspeedcube,
            wind_direction: inputdircube):
```